tests/PhpUnit/Assert.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *
+ *  (c) Nelmio <hello@nelm.io>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace PhpUnit;
+
+use PHPUnit_Framework_TestCase as PhpUnit;
+
+/**
+ * @author Théo FIDRY <theo.fidry@gmail.com>
+ */
+final class Assert extends \PHPUnit_Framework_TestCase
```

Comment:
should be renamed `PhpUnit` or `Assertor`, `Assert::assertIsA()` doesn't look good